Volume/Views/BrowserView.swift

```diff
@@ -32,8 +34,82 @@ struct BrowserView: View {
         }
     }
     
+    private var navigationTitle: String {
+        switch state {
+        case .loading:
+            return "Loading article..."
+        case .results(let article):
+            return article.publication.name
+        }
+    }
+    
+    private var isArticleLoaded: Bool {
+        switch state {
+        case .loading:
+            return true
+        case .results(_):
+            return false
+        }
+    }
+    
     // MARK: UI Components
     
+    private var navbar: some View {
+            ZStack {
+                Rectangle()
+                    .foregroundColor(.white)
+                    .shadow(color: .black.opacity(0.2), radius: 2, y: 3)
+                
+                if showToolbars {
+                    HStack {
+                        Button {
+                            presentationMode.wrappedValue.dismiss()
+                        } label: {
+                            Image("arrow-left")
+                                .resizable()
+                                .aspectRatio(contentMode: .fit)
+                                .frame(height: 24)
+                                .foregroundColor(Color.black)
+                        }
+                        
+                        Spacer()
+                        
+                        if case let .results(article) = state, let url = article.articleUrl {
+                            Link(destination: url) {
+                                Image("compass")
+                                    .resizable()
+                                    .aspectRatio(contentMode: .fit)
+                                    .frame(height: 24)
+                                    .foregroundColor(Color.black)
+                            }
+                        } else {
+                            Image("compass")
+                                .resizable()
+                                .aspectRatio(contentMode: .fit)
+                                .frame(height: 24)
+                                .foregroundColor(Color.volume.lightGray)
+                        }
+                    }
+                    .padding(.horizontal, 16)
+                    
+                    VStack(spacing: 0) {
```

Comment:
I'm not sure but if spacing is 0, can we remove `(spacing: 0)`?

Reply:
This isn't always the case but it works here.